src/vm/appdomain.cpp

```diff
@@ -8151,7 +8150,7 @@ void AppDomain::Exit(BOOL fRunFinalizers, BOOL fAsyncExit)
     LOG((LF_APPDOMAIN | LF_CORDB, LL_INFO10, "AppDomain::Domain [%d] %#08x %ls is exited.\n",
          GetId().m_dwId, this, GetFriendlyNameForLogging()));
 
-    ReJitManager::OnAppDomainExit(this);
+    CodeVersionManager::OnAppDomainExit(this);
```

Comment:
Should this call be after ETW::LoaderLog::DomainUnload to allow for rundown to access the data, or does it not matter, because rundown should only ever touch the active code?  I would expect rundown to only touch active code because you could imagine address overlap if we emitted information about every version, and that would break symbol resolution.